src/index.js

```diff
@@ -12,24 +12,43 @@ async function main() {
 
   console.log({ eventName, sha, headSha, branch, owner, repo });
   const workflow_id = core.getInput('workflow_id', { required: true });
+  const workflow_ids = workflow_id.replace(/\s/g, '').split(',');
   const token = core.getInput('access_token', { required: true });
   console.log(`Found input: ${workflow_id}`);
   console.log(`Found token: ${token ? 'yes' : 'no'}`);
   const octokit = new github.GitHub(token);
-  const { data } = await octokit.actions.listWorkflowRuns({ owner, repo, workflow_id, branch });
-  console.log(`Found ${data.total_count} runs total.`);
-  const runningWorkflows = data.workflow_runs.filter(
-    workflow => workflow.head_sha !== headSha && workflow.status !== 'completed'
-  );
-  console.log(`Found ${runningWorkflows.length} runs in progress.`);
-  for (const { id, head_sha, status } of runningWorkflows) {
-    console.log('Cancelling another run: ', { id, head_sha, status });
-    const res = await octokit.actions.cancelWorkflowRun({ owner, repo, run_id: id });
-    console.log(`Status ${res.status}`);
-  }
+
+  await promise.all(workflow_ids.map(async (workflow_id) => {
```

Comment:
I think `Promise`  should be uppercase here.

Did you test this?

Reply:
Meh.

Fixed and tested on my fork => https://github.com/mfn/cancel-workflow-action/actions

sorry